pom.xml

```diff
@@ -60,6 +60,7 @@
         <module>openmetrics</module>
         <module>packaging</module>
         <module>sample-server</module>
+        <module>secrets-plugin</module>
```

Comment:
So is the idea that we would put all plugins in one module?  We can always change this later, but I would imagine that for additional plugins we would want separate modules, and if so maybe this should be the secrets-keystore-plugin.

Reply:
No - But for airlift we could try to have all the secrets-plugin in one place -